test/extended/templates/template.go

```diff
@@ -51,7 +52,7 @@ var _ = g.Describe("[sig-devex][Feature:Templates] template-api", func() {
 				Raw: []byte(runtime.EncodeOrDie(corev1Codec, &v1.Service{
 					ObjectMeta: metav1.ObjectMeta{
 						Name:      "${NAME}-tester",
-						Namespace: "somevalue",
+						Namespace: oc.Namespace(),
```

Comment:
This line change is still a problem.  We don't want to mutate this test fixture.